### PR TITLE
Rsg type not selectable description

### PR DESCRIPTION
**Pull request by mjosborne1**
Added CodeSystem property description as a technical correction for https://jira.hl7.org/browse/FHIR-46107

### CODE REVIEW

File: input/resources/valueset-rsg-source-document-jurisdiction.xml [hunk 1]
```diff
@@ -12,11 +12,11 @@
         <system value="urn:ietf:rfc:3986"/>
         <value value="urn:oid:2.16.840.1.113883.2.3.4.2.4.33"/>
     </identifier>
-    <name value="CommonAURecordedSexOrGenderSourceDocumentType" />
```
Comment (dbojicic) [inline input/resources/valueset-rsg-source-document-jurisdiction.xml:15]: This change is not related to FHIR-46107, please revert.

File: input/resources/valueset-rsg-source-document-jurisdiction.xml [hunk 2]
```diff
@@ -12,11 +12,11 @@
...
-    <title value="Common AU Recorded Sex or Gender (RSG) Source Document Jurisdiction" />
```
Comment (dbojicic) [inline input/resources/valueset-rsg-source-document-jurisdiction.xml:16]: This change is not related to FHIR-46107, please revert.

File: input/resources/valueset-rsg-source-document-jurisdiction.xml [hunk 3]
```diff
@@ -12,11 +12,11 @@
...
+    <name value="AURecordedSexOrGenderSourceDocumentType" />
+    <title value="AU Recorded Sex or Gender (RSG) Source Document Jurisdiction" />
     <status value="draft" />
     <experimental value="false"/>
-    <description value="The Common AU Recorded Sex or Gender (RSG) Source Document Jurisdiction value set includes codes that can be used to identify the jurisdiction that issued the source document where the sex or gender value is recorded."/>
+    <description value="The AU Recorded Sex or Gender (RSG) Source Document Jurisdiction value set includes codes that can be used to identify the jurisdiction that issued the source document where the sex or gender value is recorded."/>
```
Comment (dbojicic) [inline input/resources/valueset-rsg-source-document-jurisdiction.xml:19]: This change is not related to FHIR-46107, please revert.

File: input/resources/valueset-rsg-source-document-type.xml [hunk 4]
```diff
@@ -12,11 +12,11 @@
         <system value="urn:ietf:rfc:3986"/>
         <value value="urn:oid:2.16.840.1.113883.2.3.4.2.4.32"/>
     </identifier>
-    <name value="CommonAURecordedSexOrGenderSourceDocumentType" />
-    <title value="Common AU Recorded Sex or Gender (RSG) Source Document Type" />
+    <name value="AURecordedSexOrGenderSourceDocumentType" />
+    <title value="AU Recorded Sex or Gender (RSG) Source Document Type" />
```
Comment (dbojicic) [inline input/resources/valueset-rsg-source-document-type.xml:16]: These changes are not related to FHIR-46107, please revert.

File: input/resources/valueset-rsg-source-document-type.xml [hunk 5]
```diff
@@ -12,11 +12,11 @@
...
     <status value="draft" />
     <experimental value="false"/>
-    <description value="The Common AU Recorded Sex or Gender (RSG) Source Document Type value set includes codes that can be used to represent the type of document where this sex or gender value is recorded." />
+    <description value="The AU Recorded Sex or Gender (RSG) Source Document Type value set includes codes that can be used to represent the type of document where this sex or gender value is recorded." />
```
Comment (dbojicic) [inline input/resources/valueset-rsg-source-document-type.xml:19]: This change is not related to FHIR-46107, please revert.

File: input/resources/valueset-rsg-type.xml [hunk 6]
```diff
@@ -12,11 +12,11 @@
         <system value="urn:ietf:rfc:3986"/>
         <value value="urn:oid:2.16.840.1.113883.2.3.4.2.4.31"/>
     </identifier>
-    <name value="CommonAURecordedSexOrGenderType" />
-    <title value="Common AU Recorded Sex or Gender Type" />
+    <name value="AURecordedSexOrGenderType" />
+    <title value="AU Recorded Sex or Gender Type" />
```
Comment (dbojicic) [inline input/resources/valueset-rsg-type.xml:16]: These changes are not related to FHIR-46107, please revert.

File: input/resources/valueset-rsg-type.xml [hunk 7]
```diff
@@ -12,11 +12,11 @@
...
     <status value="draft" />
     <experimental value="false"/>
-    <description value="The Common AU Recorded Sex or Gender Type value set includes codes that can be used to represent the type or category of sex or gender that is recorded." />
+    <description value="The AU Recorded Sex or Gender Type value set includes codes that can be used to represent the type or category of sex or gender that is recorded." />
```
Comment (dbojicic) [inline input/resources/valueset-rsg-type.xml:19]: This change is not related to FHIR-46107, please revert.

**Review by dbojicic — CHANGES_REQUESTED** [review]
codesystem-rsg-type.xml changes look good. 
[](url)
Please revert changes to the RSG value sets, as these are out of scope of [FHIR-46107](https://jira.hl7.org/browse/FHIR-46107) and are addressed in PR #850.

File: input/resources/valueset-rsg-type.xml (hunk 7 above)
Comment (mjosborne1) [reply input/resources/valueset-rsg-type.xml]: Reverted change

File: input/resources/valueset-rsg-type.xml (hunk 6 above)
Comment (mjosborne1) [reply input/resources/valueset-rsg-type.xml]: Reverted change

File: input/resources/valueset-rsg-source-document-type.xml (hunk 5 above)
Comment (mjosborne1) [reply input/resources/valueset-rsg-source-document-type.xml]: Reverted change

File: input/resources/valueset-rsg-source-document-jurisdiction.xml (hunk 1 above)
Comment (dbojicic) [reply input/resources/valueset-rsg-source-document-jurisdiction.xml]: Confimed.

File: input/resources/valueset-rsg-source-document-jurisdiction.xml (hunk 2 above)
Comment (dbojicic) [reply input/resources/valueset-rsg-source-document-jurisdiction.xml]: Change reverted

File: input/resources/valueset-rsg-source-document-jurisdiction.xml (hunk 3 above)
Comment (dbojicic) [reply input/resources/valueset-rsg-source-document-jurisdiction.xml]: Change reverted

File: input/resources/valueset-rsg-source-document-type.xml (hunk 4 above)
Comment (dbojicic) [reply input/resources/valueset-rsg-source-document-type.xml]: Change reverted

File: input/resources/valueset-rsg-source-document-type.xml (hunk 5 above)
Comment (dbojicic) [reply input/resources/valueset-rsg-source-document-type.xml]: Change reverted

File: input/resources/valueset-rsg-type.xml (hunk 6 above)
Comment (dbojicic) [reply input/resources/valueset-rsg-type.xml]: Change reverted

File: input/resources/valueset-rsg-type.xml (hunk 7 above)
Comment (dbojicic) [reply input/resources/valueset-rsg-type.xml]: Change reverted

**Review by dbojicic — APPROVED** [review]

**Comment by dbojicic** [comment]
No issues found; @dt-r, @brettesler-ext - please merge.
